src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Person } from '../../types';
+import { BodyTable } from '../BodyTable';
+
+type Props = {
+  people: Person[],
+  selectedSlug: string | undefined,
+};
+
+const tableHead = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+export const PeopleTable: React.FC<Props> = ({ people, selectedSlug }) => {
+  // const location = useLocation();
+  // console.log(location);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {tableHead.map(currentColumn => (
+            <th key={`${currentColumn}_Date.now()`}>{currentColumn}</th>
```

Comment:
Be careful, because now you are not calling the function, but recording its call as a string.
```suggestion
            <th key={`${currentColumn}_${Date.now()}`}>{currentColumn}</th>
```